index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cars</title>
+	<!-- Latest compiled and minified CSS -->
+<!-- Latest compiled and minified CSS -->
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="css/quiz.css">
+
+<!-- Latest compiled and minified JavaScript -->
+
+ 
+  
+</head>
+<body>
+<nav class="navbar navbar-light bg-faded">
+  <form class="form-inline float-xs-left">
+    <input id="inputText" type="text">
+    <button id="submitBtn" type="submit">Submit</button>
+  </form>
+</nav>
+ 
+	<div class="container" id="cars-grid">
+		
+	</div>
+
+<script
+  src="dist/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
You have installed Bootstrap and moved it to the `dist` folder but you're not even using it. This is a link to a version on a remote server.